sync-partials.js

```diff
@@ -0,0 +1,28 @@
+import glob from "glob";
+import fs from "fs";
+
+// use this comment syntax to include markdown partials between the include and end tags
+// <!-- INCLUDE path-to-partial.md -->
+// <!-- END -->
+
+const includeRegex = /(\<\!\-\- INCLUDE (.*.md) -->)(.|\n)*(<!-- END -->)/g;
```

Comment:
Why have the end tag?

Reply:
we need to know where the content ends so we can overwrite *idempotently*